### ISSUE TITLE
React Native have to double click for onPress to work inside Tab

### ISSUE THREAD

**Issue by samjabeen** [GeekyAnts/NativeBase#1601]
Hi! 

Thanks in advance!

I'm using React Native with the Native Base library. Problem is double click needed onPress event to fire on Native Base' Button inside Tab  when the keyboard is open.

Sample code
```
	<Tabs>
		<Tab>
			
			<Content keyboardShouldPersistTaps='always' keyboardDismissMode='on-drag'>
				<Input
					keyboardType = 'numeric'
					keyboardType={Platform.OS === 'ios' ? 'number-pad' : 'numeric'}
                />
				<Button rounded  danger onPress={this.handleUpdateClick}>
                        <Text>Update</Text>
				</Button>
			<Content>
	
		</Tab>
		...
	
	</Tabs>
```
	"react-native": "0.47.1",
	"native-base": "1.1.0"

Expected behaviour
	Need an onPress event to fire on Native Base' Button inside Tab  when the keyboard is open
Actual behaviour
	double click needed for onPress o fire on Native Base' Button inside  Tab when the keyboard is open

**Comment by SupriyaKalghatgi** [comment]
Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)

**Comment by samjabeen** [comment]
@SupriyaKalghatgi thanks for the update.
